### PR TITLE
Fix IDNCLOUD-280: Make application type options more meaningful for the end user.

### PR DESCRIPTION
**Pull request by omindu**

### CODE REVIEW

File: jaggeryapps/admin/serviceproviders/editsp.jag [hunk 1]
```diff
@@ -141,17 +141,16 @@ include("../templates/leftpane.jag");
               <div class="btn-group" id="custom-app-dropdown" style="display: none;">
                   <a class="btn btn-default dropdown-toggle" data-toggle="dropdown" href="#" id="app-type-dropdown">Select App Type<span
                           class="caret"></span></a>
-                  <ul class="dropdown-menu">
-                      <li><a id="agenttype" href="#">Agent</a></li>
-                      <li><a id="proxytype" href="#">Proxy</a></li>
+                  <ul class="dropdown-menu" id="app-type-drop-down">
+                      <li><a id="agenttype" href="#">Standards-Based Federation</a></li>
```
Comment (malithie) [inline jaggeryapps/admin/serviceproviders/editsp.jag]: Should this be standards-based or standard-based ? Can you please verify with someone from doc team

File: jaggeryapps/admin/serviceproviders/editsp.jag [hunk 2]
```diff
@@ -1136,8 +1141,11 @@ $("#custom-app-dropdown .dropdown-menu li a").click(function () {
             $("#security-accordion").show();
             $("#cladimconfig-header").remove();
             $("#inbound-auth-header").hide();
+            //set app type description
+            $("#app-type-desc-span").text("Application should be able to generate standard SSO requests and process responses.");
+            $("#app-type-desc-url").attr('href', 'https://docs.wso2.com/display/IdentityCloud/Configuring+Agent-based+SSO');
```
Comment (malithie) [inline jaggeryapps/admin/serviceproviders/editsp.jag]: Better to move these documentation links to some conf file and refer from that here.  So if the documentation links got changed it's easy to locate them

**Comment by malithie** [comment]
please get the text in html reviewed with docs team

**Review by malithie — CHANGES_REQUESTED** [review]

**Review by malithie — APPROVED** [review]

File: jaggeryapps/admin/serviceproviders/editsp.jag (hunk 1 above)
Comment (omindu) [inline jaggeryapps/admin/serviceproviders/editsp.jag]: standards-based is the correct word.

**Pull request merged**